yocto_standalone_ci.adoc

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2023 Savoir-faire Linux, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+Setup Yocto standalone CI
+=========================
+
+== Prerequisites
+
+* A machine that meets SEAPATH requirements
+* An USB drive
+
+== USB driver setup
+
+* Generate the SEAPATH flasher image (cqfd flasher flavor ) with SEAPATH_AUTO_FLASH enable
```

Comment:
Generate the SEAPATH flasher image with `cqfd -b flasher`. Make sure  `SEAPATH_AUTO_FLASH` is enable in `seapath.conf`.